packages/unittest_runner/unittest_runner/run_tests.py

```diff
@@ -0,0 +1,18 @@
+import unittest
+from .validation_runner import ValidationTestResult
+
+# Run the tests in the given file pattern and display the results to the user.
+# Validation tests use the ValidationTestResult class to display only the short description of the test.
+def run_validation_tests(file_pattern):
+  run_tests(file_pattern, ValidationTestResult)
+
+# Run the tests in the given file pattern and display the results to the user.
+# Student tests use the standard TextTestResult class to display the test name and short description.
+def run_student_tests(file_pattern):
+  run_tests(file_pattern, unittest.TextTestResult)
+
+def run_tests(file_pattern, resultclass):
+  loader = unittest.TestLoader()
+  test_suite = loader.discover('.', file_pattern)
```

Comment:
Does this mean if student code were to modify `PWD` the test loader would end up looking for test files somewhere other than the root of their project? Not a big concern, but just trying to make sure I'm tracking.

Reply:
Hmm good question. This runs before any student code is run, so I think that case would only occur if they ran their code before running tests, and the working directory persisted between runs, which it might. 

FWIW we so far aren't having students run their own unit tests, but I wonder if changing the working directory would interfere with us finding the validation test file on a subsequent run. I'll file a ticket to make sure we always start from the correct working directory.

Reply:
https://codedotorg.atlassian.net/browse/CT-703